R/misc.R

```diff
@@ -21,13 +21,21 @@ get_rhs_vars <- function(formula, data, no_lhs = FALSE) {
   ## or should it? what about Y ~ log(x)?
   ## Answer: when called from `form2args`, the function
   ## `inline_check` stops when in-line functions are used.
-  data_info <- attr(model.frame(formula, data[1, ]), "terms")
```

Comment:
Does the comment above still apply?

Reply:
yes! we can do the simpler things because we are certain that there isn't going to be inline functions anymore